formspree/static/scss/dashboard.scss

```diff
@@ -36,13 +36,13 @@
   }
 
   .delete {
-    color: #D9534F;
-    border-color: #D9534F;
+    color: $red;
+    border-color: $red;
   }
 
   .delete:hover {
     transition: color 0.3s ease-in-out;
-    color: #d1332e;
+    color: $warning-orange;
```

Comment:
@rohitdatta I changed these variables to make them use the same colours that are allegedly being used in other places, but I think it doesn't work. Does it? Well, I now my `.red` class down there doesn't work.

@colevscode does the `scss` preprocessor actually know where to look for these variables, since they aren't imported in this file? Am I using the wrong tool here? (I don't even know where did I get this `scss` program I have here.)